src/DDTrace/ID.php

```diff
@@ -14,6 +19,19 @@ public static function generate()
          * will do. And since all integers in PHP are signed, an int
          * between 1 & PHP_INT_MAX will be 63-bit.
          */
-        return (string) mt_rand(1, PHP_INT_MAX);
+        $first31bits = mt_rand(0, mt_getrandmax()) << 32;
+        $second31bits = mt_rand(0, mt_getrandmax()) << 1;
+        $random1or0 = mt_rand(0, 1);
+        return (string) ($first31bits | $second31bits | $random1or0);
```

Comment:
Nice workaround @labbati! For the future PR, we'll look into possibly supporting 32-bit architectures as well. :)